src/tags/bitwise.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @Author: RagingLink 
+ * @Date: 2020-08-17 14:03:17
+ * @Last Modified by: RagingLink
+ * @Last Modified time: 2021-05-20 23:03:10
+ *
+ * This project uses the AGPLv3 license. Please read the license file before using/adapting any of the code.
+ */
+
+ const Builder = require('../structures/TagBuilder');
+ let operators = {
+     '&': (a, b) => a & b,
+     '|': (a, b) => a | b,
+     '^': (a, b) => a ^ b,
+     '~': (a) => ~a,
+     '<<': (a, b) => a << b,
+     '>>': (a, b) => a >> b,
+     '>>>': (a, b) => a >>> b
+ }
+ 
+ module.exports =
+     Builder.AutoTag('bitwise')
+         .acceptsArrays()
+         .withArgs(a => [a.require('operator'), a.require('value'), a.optional('othervalue', true)])
+         .withAlias('bitmath')
+         .withDesc(`Performs a bitwise operation on the provided value(s). Supplied values must be binary numbers.\nValid operators are: \`${Object.keys(operators).join(', ')}\``)
+         .withExample(
+             '{bitwise;^;1000;1001}',
+             '1000'
+         ).whenArgs('0-1', Builder.errors.notEnoughArguments)
+         .whenDefault(async (subtag, context, args) => {
+             if (!operators.hasOwnProperty(args[0]))
+                 return Builder.errors.invalidOperator(subtag, context);
+             let operator = operators[args[0]]
```

Comment:
I'd rather call the variable `operation` as the object returns an operation based on the provided operator.